src/redux/modules/lists.js

```diff
@@ -93,13 +97,36 @@ const onRemoveItem = (state, action) => {
       })),
   };
 };
+const onRemoveItems = (state, action) => {
```

Comment:
@edoshor All changes here are part of 
https://github.com/Bnei-Baruch/mdb-admin/pull/37/files#diff-beec26dd536674367ac8a2c781147420R210